deltachat-jsonrpc/src/api/types/webxdc.rs

```diff
@@ -72,7 +75,11 @@ impl WebxdcMessageInfo {
             source_code_url: maybe_empty_string_to_option(source_code_url),
             internet_access,
             self_addr,
-            send_update_interval,
+            send_update_interval: send_update_interval
```

Comment:
For the future i suggest to add the `_ms` suffix, otherwise one can think it's in seconds. Moreover we have other intervals in API which are indeed in seconds